instrumentation_statsd_sink.go

```diff
@@ -308,7 +308,7 @@ func (s *StatsDSink) writeStatsDMetric(b []byte) {
 
 	lenUdpBuf := s.udpBuf.Len()
 
-	if (lenb + lenUdpBuf) > maxUdpBytes {
+	if lenb+lenUdpBuf > maxUdpBytes {
```

Comment:
Well, if this was a gofmt suggestion, then for sure will be easier not to fight with it. 👍